src/framework/uicomponents/view/dialogview.cpp

```diff
@@ -118,8 +121,9 @@ void DialogView::updateGeometry()
     setContentWidth(dlgRect.width());
     setContentHeight(dlgRect.height());
 
-    //! NOTE ok will be if they call accept
-    setErrCode(Ret::Code::Cancel);
```

Comment:
I agree that `updateGeometry` is not the best place for something like this. But doesn't it need to be moved to a different place, instead of simply deleted?